docs/modules/player-lifecycle/heartbeating.md

```diff
@@ -0,0 +1,38 @@
+<%(TOC)%>
+# Heartbeating
+
+Heartbeating is a technique used to regularly checking-in on the health of connected client-workers and ensuring that unresponsive or disconnected clients are periodically removed from the game world.
+
+## Why is it useful?
```

Comment:
I thought we don't use question marks for headers? @jamiebrynes7

Reply:
I don't think we do